source/blog/2016-02-17-ADB-1-7.html.md

```diff
@@ -0,0 +1,36 @@
+---
+author: Lalatendu Mohanty
+layout: post
+comments: true
+title: Releasing ADB-Atomic Developer Bundle 1.5
+date: 2016-02-17 11:50 UTC
+tags:
+- vagrant
+- Docker
+- Developer tools
+- openshift
+- kubernetes
+- centos
+categories:
+- Blog
+---
+Project Atomic is pleased to announce the release of the Atomic  Developer Bundle (ADB) 1.7.0, a Vagrant box that provides a ready-to-use  development environment for container applications. With the ADB,  developers can dive right into producing complex, multi-container  applications. The ADB is useful on Microsoft Windows, Apple’s OS X and  GNU/Linux distributions.
+
+The ADB provides a container development ecosystem that ensures that  developers are able to work with Linux containers even if their base operating system does not support it.
+The ADB provides a quick and easy way to setup multi-container  applications that take advantage of management and/or orchestration  solutions, such as OpenShift or Kubernetes. The ADB also supports [Nulecule](http://www.projectatomic.io/docs/nulecule/) and  [Atomic App](http://www.projectatomic.io/docs/atomicapp/).  These projects provide a structure for packaging, distributing, and deploying multi-container applications.
+
+The 1.7.0 release adds several new features, including:
+ * OpenShift is now managed as a systemd unit file with full functionality [start/stop/restart].
+ * We have a new cli i.e. sccli inside the box which can help users move between Kubernets and OpenShift setup.
+ * We have a new Vagrant plugin i.e. [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager/) to manage services inside the ADB Vagrant box .  This is the recomended plugin inplace of vagrant-adbinfo now.
```

Comment:
Suggested:

* We have a new vagrant plugin, vagrant-service-manager, that manages the services inside of the ADB.  This plugin is recommended for use with the ADB and fully replaces vagrant-adbinfo.